corehq/apps/app_manager/templates/app_manager/partials/modules/module_view_settings.html

```diff
@@ -255,6 +255,18 @@ <h4 class="panel-title panel-title-nolink">{% trans "Advanced" %}</h4>
               {% include "app_manager/partials/session_endpoints.html" with module_or_form=module %}
             {% endif %}
 
+            {% if request|toggle_enabled:'USH_EMPTY_CASE_LIST_TEXT' %}
+            <div class="form-group">
+              <label for="description" class="{% css_label_class %} control-label">
+                {% trans "Empty Case List Text" %}
+                <span class="hq-help-template" data-title="{% trans "Empty Case List Text" %}"
+                      data-content="{% trans_html_attr "Define text displayed when case list has no results." %}">
```

Comment:
```suggestion
                <span class="hq-help-template" data-title="{% trans "Empty Case List Text" %}"
                      data-content="{% trans_html_attr "Define text displayed when case list has no results." %}"
                ></span>
```

Reply:
Corrected in commit [597747c](https://github.com/dimagi/commcare-hq/pull/32491/commits/597747cc6e750c87ba000f716d888131cdaa06cb)